lib/auth_mvp_web/controllers/google_auth_controller.ex

```diff
@@ -0,0 +1,15 @@
+defmodule AuthMvpWeb.GoogleAuthController do
+  use AuthMvpWeb, :controller
+
+  @elixir_auth_google Application.get_env(:auth_mvp, :elixir_auth_google) || ElixirAuthGoogle
+
+  @doc """
+  `index/2` handles the callback from Google Auth API redirect.
+  """
+  def index(conn, %{"code" => code, "state" => client}) do
+    {:ok, token} = @elixir_auth_google.get_token(code, conn)
+    {:ok, profile} = @elixir_auth_google.get_user_profile(token.access_token)
```

Comment:
We can add a function which will get the token and the user profile, but I think I'll still keep the `get_token` and `get_user_profile`

Reply:
Yeah, we will definitely keep these two functions so that people can use the code/token _and_ get the user's profile. But I'm a _huge_ fan of having _consistent_ APIs that look like they were written by one person. 💭